browserslist-stats.json

```diff
@@ -0,0 +1 @@
":0.0009641331046213515,"23":0.0011765692124192763,"24":0,"25":0.014314925417767863,"26":0.0026309394888819927,"27":0.002222408512347522,"28":0.05261878977763986,"29":0.002026313643610976,"30":0.005556021280868804,"31":0.01843291766123533,"32":0.003807508701301269,"33":0.003448001441950935,"34":0.07178706319663723,"35":0.015851001889537475,"36":0.008742562897837678,"37":0.013628593377189949,"38":0.14009344247320077,"39":0.021848236625063504,"40":0.02884228694333365,"41":0.02547599169668961,"42":0.04740593451706001,"43":0.05987429992089206,"44":0.16759574781350137,"45":0.48835792934930655,"46":0.04531425591720352,"47":0.06119794028486375,"48":0.04497108989691456,"49":0.7162691905383572,"50":0.24157253704436338,"51":0.8787011068084629,"52":0.07884647847115289,"53":0.16650088479638897,"54":0.1889700885057849,"55":0.6444984685807813,"56":0.5867321884988072,"57":0.8187124182141412,"58":6.437859905577052,"59":39.71033846464581,"60":0.1427734056792669,"61":0.058027739906956255,"62":0.0012582754077261706,"63":0},"safari":{"4":0.0002124361077979249,"5":0.011340819908596913,"6":0.01640660401762435,"7":0.09693623011209927,"8":0.2757747503998293,"9":0.9620087435433722,"10":12.677353510188027,"11":0.11082628331427129,"3.1":0,"3.2":0,"5.1":0.026897679495029567,"6.1":0.02279602849062348,"7.1":0.0009151093874372149,"9.1":0.7454056197847957,"10.1":4.8317612244294335,"TP":0},"opera":{"15":0,"16":0,"17":0,"18":0,"19":0,"20":0,"21":0,"22":0,"23":0,"24":0,"25":0,"26":0.00006536495624551535,"27":0,"28":0,"29":0,"30":0,"31":0,"32":0,"33":0,"34":0.00006536495624551535,"35":0.00014707115155240954,"36":0.00506578410902744,"37":0,"38":0.0001307299124910307,"39":0,"40":0.00006536495624551535,"41":0.00006536495624551535,"42":0.002434844620145447,"43":0.0007843794749461842,"44":0.005147490304334334,"45":0.013334451074085131,"46":0.07417288409959855,"47":0,"48":0.0007190145187006689,"10.0-10.1":0,"11.5":0,"12.1":0.002157043556102007},"ios_saf":{"8":0.0004902371718413651,"11":0.003202882856030252,"3.2":0,"4.0-4.1":0,"4.2-4.3":0.0002124361077979249,"5.0-5.1":0.0004902371718413651,"6.0-6.1":0.001519735232708232,"7.0-7.1":0.004019944809099194,"8.1-8.4":0.007778429793216328,"9.0-9.2":0.014658091438056816,"9.3":0.03140786147597013,"10.0-10.2":0.12878530504272662,"10.3":0.5630373918598078},"op_mini":{"all":0.13708665448590707},"android":{"3":0.0001307299124910307,"4":0.01222324681791137,"56":0.5115951712945873,"2.1":0,"2.2":0.00027780106404344025,"2.3":0.0025165508154523413,"4.1":0.01083424149769417,"4.2-4.3":0.018759742442462906,"4.4":0.028989358094886057,"4.4.3-4.4.4":0},"bb":{"7":0.006601860580797051,"10":0.010621805389896245},"op_mob":{"12":0.005359926412132259,"37":0.005359926412132259,"12.1":0.005359926412132259},"and_chr":{"59":0},"and_ff":{"54":0},"ie_mob":{"10":0,"11":0},"and_uc":{"11.4":0},"samsung":{"4":0,"5":0},"and_qq":{"1.2":0},"baidu":{"7.12":0}},"meta":{"start_date":"2017-06-27","end_date":"2017-07-27"},"uid":"custom.9041800|undefined","unknownPerc":1.304194289488649}
```

Comment:
Did you use [Browserslist-GA](https://github.com/dmfrancisco/browserslist-ga) to generate this?

Reply:
I can't find it anymore, but a guide revealed a variable I could console log while viewing our stats through caniuse.com.